lib/AST/ClangTypeConverter.h

```diff
@@ -34,6 +34,7 @@ class ClangTypeConverter :
 
   llvm::DenseMap<Type, clang::QualType> Cache;
   llvm::DenseMap<const clang::Decl *, swift::Decl *> ReversedExportMap;
+  llvm::DenseMap<swift::Decl *, const clang::Decl *> ReversedExportMapBackwards;
```

Comment:
I know `ReversedExportMap` doesn't have one yet, but can you document what this map does? `Reversed[...]Backwards` is a double-negative that's going to trip up folks reading this later.